news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "Hacktoberfest 2020 is coming"
+subtitle: "Bring the lights on open source projects"
+date:  2020-10-01 09:10:11
+authors: [LouiseBonnard]
+icon: icon-github-alt
+image: /assets/images/2020/09/Hacktoberfest-2020.png
+tags: [community, contribution, github, contribute]
+---
+
+Open source has its own tradition when fall starts cooling the atmosphere around, Hacktoberfest! Beer is not flowing like water but code is, and it creates more value than any bunch of slightly tipsy people. ;-)
+
+Curated by the American cloud hosting company [DigitalOcean](https://www.digitalocean.com), with Intel and DEV, [Hacktoberfest](https://hacktoberfest.digitalocean.com) happens online every year in October to celebrate open source. It’s pretty simple: for one month, people are encouraged to engage and create at least 4 pull requests for open source projects on GitHub.
+
+As the open source community spreads universal values, everyone is welcome to learn and share knowledge, and therefore stands together to build beneficial technologies. All those individual contributions create one great collective effort that makes this possible.
+
+As a result, it is probably the best place to start involving in the projects you want to support. And the numbers speak for themselves since last year, no less than 483,127 pull requests have been submitted on 154,767 participating repositories!
+
+![Hacktoberfest 2020](/assets/images/2020/09/Hacktoberfest-2020.png)
+
+
+## Get acquainted with the Hacktoberfest challenge
+
+Every contributor achieving this goal of 4 contributions merged will be sent a pack of nice goodies including a tee-shirt and many wonderful stickers. It is the perfect occasion to start a collection! Looking for a more sustainable gift? An environment-friendly option, the ‘pick trees, not tees’ option, will plant a tree instead of delivering you goodies. ;-)
```

Comment:
"the 'pick" est-il intentionnel ?

Reply:
Yep, it is a goodie option called _pick trees, not tees_. It lets the participants choose between the classic Hacktoberfest tee-shirt and a more green goodie: a planted tree! 🌳